configuration/concepts/lab_tests.csv

```diff
@@ -126,15 +126,16 @@ ca28bdb2-c032-4b2f-b33f-fd73de27b271,,,Widal,Widal,Widal,Widal,,,LabTest,Text,,,
 7f66be56-1d90-46f4-b849-1eeeba4fae21,,,Zeihl,Zeihl,Zeihl,Zeihl,,,LabTest,Text,,,,,,,,,,,
 533e1832-00e0-4228-a61c-6df1d68da279,,,Direct Bilirubin,Bilirubine directe,Direct Bilirubin,Bilirubine directe,,,LabTest,Numeric,,,,yes,mg/dL,0.1,0.5,,,,
 6d08b363-eeeb-4fe6-9377-dcf9a6443d0d,,,Indirect Bilirubin,Bilirubine indirecte,Indirect Bilirubin,Bilirubine indirecte,,,LabTest,Text,,,,,,,,,,,
-dbfa32b2-70d4-4a90-9ec0-4e95f500c1a8,,CIEL:1000,Blood,Blood,Blood,Blood,,,Sample,N/A,,,"Acide Urique;Albumine dans le sang;ALT (SGPT);Amylase;ASO (Antistreptolysine O);AST (SGOT);Azote de l'Urée (BUN);BHCG;Bilirubine Totale;Phosphatase Alcaline (ALKP);Lactic Acid Dehydrogenase;Bilan Hépatique (AST-ALT-TBil-ALKP-LDH);Cholesterol;Triglyceride;Very low-density lipoprotein cholesterol;Low-density lipoprotein cholesterol;High-density lipoprotein cholesterol;Bilan Lipidique (Chol-Trig-HDLC-LDL-VLDL);Creatinine (CRSC);Urée;Bilan Renal (BUN-Creat-Urée-Acide Urique);T3;T4;TSH;Bilan Thyroïdien;Calcium (Ca);Glycémie;Chimie Sanguine (Glycemie-BUN-Urée-Créatine) ;Chlore (Cl);Potassium (K);Sodium (Na);CO₂;Ionogramme (Na, K, Cl, CO₂);Créatinine Kinase (CK);Créatinine Kinase MB;CRP;ECBC des Liquides (TP-ALB-LDH-GLU-WBC-RBC-Diff…);Fer (Fe);Filaire test Diurne;Filaire test Nocturne;Globules Blanches;Globules Rouges;Glycemie Plasmatique;Glycemie Capillaire;Groupe Sanguin (ABO, Rh);Hematocrite;Taux de leucocytes;Taux d’hématies;Hemoglobine;Neutrophiles;Eosinophiles;Basophiles;Lymphocites;Monocytes;Taux de reticulocytes;Mean Corpuscular Volume;Mean Corpuscular Hemoglobin Concentration;Mean Corpuscular Hemoglobin;Plaquettes Sanguines;Hemogramme Complet;Hemoglobine Glycolysilé(HBA1C);H-Pilori test;Lame Peripherique;Lipase;Magnesium (Mg);Malaria test;Panel Cardiaque (CKMB-MYOGLOBINE-TREPONINE);Proteine Serique;Proteine Totale;PSA;PT-INR;RPR;Sickling test;Widal;Vitesse de Sedimentation (VS);Zeihl;Bilirubine directe;Bilirubine indirecte",,,,,,,,
+e9120969-90c8-4662-8a3c-d33c2d46277a,,,D-Dimer Exclusion,D-Dimer Exclusion,D-Dimer Exclusion,D-Dimer Exclusion,,,LabTest,Text,,,,,,,,,,,
```

Comment:
Is this Lab test going to be Free text? I just want to confirm you are sure that's gona be always true once merged.

Reply:
Yes @Ruhanga it is a text field -->https://mekomsolutions.slack.com/archives/G8ZM2J4H4/p1716207865569369?thread_ts=1715868611.023309&cid=G8ZM2J4H4